### DISCUSSION TITLE
Case-insensitive search by tag

### DISCUSSION THREAD

**Discussion by ohylli** [apache/iotdb#13350] [Q&A] [answered]
When querying for a timeseries by tag contents, is it possible to make the search case-insensitive. For example a query like this one:

```
show timeseries root.mydb.** where TAGS(description) contains 'temperature'
```

Would not work if temperature was the first word of a sentence and thus the first letter would be capitalized.

**Comment by CritasWang** [comment, marked as answer]
This operation is currently not feasible.
You can try modify it.
